src/views/ScorePage/ScorePage.js

```diff
@@ -69,6 +62,26 @@ function nicknameValidation() {
   }
 }
 
+function saveQuizScore() {
+  const quizScores = JSON.parse(localStorage.getItem('quizScores')) || [];
+  const recentUserScore = Answers.reduce((a, b) => a + b.getScore(), 0);
```

Comment:
duplicate from line 10. Extract it to separate function (ex. getCurrentScore(Answers))